nuxt.config.js

```diff
@@ -292,14 +292,17 @@ module.exports = {
 
   // Proxy: https://github.com/nuxt-community/proxy-module#options
   proxy: {
-    '/k8s':       proxyWsOpts(api), // Straight to a remote cluster (/k8s/clusters/<id>/)
-    '/api':       proxyWsOpts(api), // Management k8s API
-    '/apis':      proxyWsOpts(api), // Management k8s API
-    '/v1':        proxyWsOpts(api), // Management Steve API
-    '/v3':        proxyWsOpts(api), // Rancher API
-    '/v3-public': proxyOpts(api), // Rancher Unauthed API
-    '/api-ui':    proxyOpts(api), // Browser API UI
-    '/meta':      proxyOpts(api), // Browser API UI
+    '/k8s':               proxyWsOpts(api), // Straight to a remote cluster (/k8s/clusters/<id>/)
+    '/api':               proxyWsOpts(api), // Management k8s API
+    '/apis':              proxyWsOpts(api), // Management k8s API
+    '/v1':                proxyWsOpts(api), // Management Steve API
+    '/v3':                proxyWsOpts(api), // Rancher API
+    '/v3-public':         proxyOpts(api), // Rancher Unauthed API
+    '/api-ui':            proxyOpts(api), // Browser API UI
+    '/meta':              proxyOpts(api), // Browser API UI,
+    '/assets':            proxyOpts(api),
+    '/translations':      proxyOpts(api),
+    '/verify-auth-azure': proxyOpts(api),
```

Comment:
Hmm this is a big problem..  `/verify-auth-azure` is an Ember page you're proxying through.  There needs to be a Vue equivalent (e.g. github uses `pages/auth/verify.vue`) so this can work without Ember being involved.

But the `https://<rancher>/verify-auth-azure` URL is also part of the auth config that we give to the user to configure in Azure, so it can't just be changed depending on where you're tying to login to (as it can for GitHub).

Or maybe it can, [redirect_uri](https://docs.microsoft.com/en-us/azure/active-directory/develop/v2-oauth2-implicit-grant-flow) _says_ it must exactly match, but will it take a query param, or match a sub-path if it were set to just `https://<rancher>/`?

Assuming that doesn't work I think we will need to store a bit somewhere (query param, or part of the `state` input) for vue vs ember.  Then have the backend look at that and respond to `GET /verify-auth-azure` with one index.html or the other depending on what it says.

Reply:
> Hmm this is a big problem.. /verify-auth-azure is an Ember page you're proxying through. There needs to be a Vue equivalent (e.g. github uses pages/auth/verify.vue) so this can work without Ember being involved.

Is there a reason we can't wait to make this change for when ember is no longer involved?

> But the https://<rancher>/verify-auth-azure URL is also part of the auth config that we give to the user to configure in Azure, so it can't just be changed depending on where you're tying to login to (as it can for GitHub).

> But the https://<rancher>/verify-auth-azure URL is also part of the auth config that we give to the user to configure in Azure, so it can't just be changed depending on where you're tying to login to (as it can for GitHub).

Initial testing seems to indicate configuring Azure with `https://<rancher>/` will allow sub-paths so we could provide a new path and page specifically for dashboard.

Does this help us though? We will have some clients that have already configured `https://<rancher>/verify-auth-azure`.

Reply:
It's just a landmine that's going to be long forgotten when we think everything is migrated.  And some modes of packaging rancher might have auth but not the ember UI fairly soon.

I wasn't actually expecting that to work because the docs say exact.. yeah there's still an existing problem so you can't just switch, but a suffix to `/verify-auth-azure/<something>` might still be useful.  e.g. we could pass in `redirect_url=/verify-auth-azure/dashboard/auth/verify` for Vue and have the API server intercept that and return 
a redirect to the subpath with the prefix stripped off